src/components/hmi_message_handler/test/CMakeLists.txt

```diff
@@ -52,6 +52,10 @@ set(LIBRARIES
     ConfigProfile
 )
 
+if(${OS} STREQUAL "qnx")
```

Comment:
@BSolonenko in some places you are using `if (${CMAKE_SYSTEM_NAME} MATCHES "QNX")` 
Try to be consistent and use the same check in all places

Reply:
@AKalinich-Luxoft  fixed in https://github.com/Ford-Luxoft/sdl_core/pull/17/commits/bd843e40d22a303560a02c26487ca620a7f8dfa5